.circleci/config.yml

```diff
@@ -33,7 +199,40 @@ jobs:
           command: ./scripts/deploy.sh
 
 workflows:
-  version: 2
+  build-test:
+    jobs:
+      - lint
+      - test-machine:
+          name: "test-client"
+          test_packages: "./client/..."
+      - test-machine:
+          name: "test-nomad"
+          test_packages: "./nomad/..."
+      - test-container:
+          name: "test-api"
+          test_packages: "./api/..."
+      - test-container:
+          name: "test-devices"
+          test_packages: "./devices/..."
+      - test-machine:
+          name: "test-other"
+          exclude_packages: "./api|./client|./drivers/docker|./drivers/exec|./drivers/rkt|./drivers/shared/executor|./nomad|./devices"
+      - test-machine:
+          name: "test-docker"
+          test_packages: "./drivers/docker"
+      - test-machine:
+          name: "test-exec"
+          test_packages: "./drivers/exec"
+      - test-machine:
+          name: "test-shared-exec"
+          test_packages: "./drivers/shared/executor"
+      - test-rkt
+      # - build-deps-image:
+      #     context: dani-test
+      #     filters:
+      #       branches:
+      #         only: dani/circleci
```

Comment:
Are the rkt tests still a WIP?

Reply:
rkt tests are fine - auto building the build image on CI is a wip bc we don't have an org yet